homework9_java8/src/BirthdaySorter.java

```diff
@@ -0,0 +1,55 @@
+import java.io.*;
+import java.nio.charset.StandardCharsets;
+import java.time.LocalDate;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.Comparator;
+import java.util.List;
+
+public class BirthdaySorter {
+    List<Person> personList = new ArrayList<>();
+    List<String> orderedList = new ArrayList<>();
+
+    public void getPersonList(String path) throws IOException {
+        BufferedReader br = new BufferedReader(new InputStreamReader(
+                new FileInputStream(path), StandardCharsets.UTF_8));
+        String t;
+        while ((t = br.readLine()) != null) {
+            String[] person = t.split(",");
+            String name = person[0];
+
+            String[] date = person[1].split("-");
+            LocalDate birthdate = LocalDate.of(Integer.parseInt(date[0]), Integer.parseInt(date[1]), Integer.parseInt(date[2]));
+
+            personList.add(new Person(name, birthdate));
+        }
+        this.personList = personList;
+    }
+
+    public void getOrderedList(List<Person> lista, int month) {
```

Comment:
Ideea temei era sa se implementeze aceasta logica folosind notiunile introduse in java 8 (streamuri si expresii lambda)